src/hdtAssetValidator.cpp

```
@@ -0,0 +1,989 @@
+				}
+			}
+		}
+
+		return result;
+	}
+
+	// ---- Phase 2: NIF discovery ----
+
+	// Scan the game data directory for NIF files that contain physics data.
+	// Two-step: serial directory walk to collect paths, then parallel binary scan.
+	static std::vector<PhysicsAsset> discoverPhysicsNIFs()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)